pkg/api/routers/router.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
Do we need to change copyright time to 2019?

Reply:
I've seen mixed advice about whether it really matters or not, but since I did update it elsewhere and missed doing it here, I do think I should change that to be consistent.